embassy-stm32/src/rcc/f4.rs

```diff
@@ -28,11 +28,71 @@ pub struct Config {
     pub sys_ck: Option<Hertz>,
     pub pclk1: Option<Hertz>,
     pub pclk2: Option<Hertz>,
+    pub plli2s: Option<Hertz>,
 
     pub pll48: bool,
 }
 
-unsafe fn setup_pll(pllsrcclk: u32, use_hse: bool, pllsysclk: Option<u32>, pll48clk: bool) -> PllResults {
+#[cfg(any(
+    feature = "stm32f411",
```

Comment:
Is there a way to select all of the chip variants without adding them all here? Something like a regex?

Reply:
build.rs defines some cfgs with cut down versions of the chip name https://github.com/embassy-rs/embassy/blob/master/embassy-stm32/build.rs#L837

so this works:

```rust
#[cfg(any(
    stm32f411,
    stm32f412,
    stm32f413,
    stm32f423,
    stm32f446,
))]
```